sql/catalyst/src/main/scala/org/apache/spark/sql/catalyst/plans/logical/v2Commands.scala

```diff
@@ -347,7 +347,7 @@ case class DescribeRelation(
     partitionSpec: TablePartitionSpec,
     isExtended: Boolean) extends Command {
   override def children: Seq[LogicalPlan] = Seq(relation)
-  override def output: Seq[Attribute] = DescribeCommandSchema.describeTableAttributes()
+  override val output: Seq[Attribute] = DescribeCommandSchema.describeTableAttributes()
```

Comment:
Shall we follow others like `ShowTables` and put the `output` as a parameter? Then it's more stable and the output won't after copy/transformation.

Reply:
> Shall we follow others like ShowTables and put the output as a parameter?

Put the `output` as a parameter doesn't solve any problems.

> Then it's more stable ...

I would say "super stable" even it is not necessary, see https://github.com/apache/spark/pull/31675

Reply:
> Then it's more stable and the output won't change after copy/transformation.

ok. `val output` will be re-initialized per every `.copy()`. I will make it as a case class parameter.